src/pages/api/citations/[msid]/bibtex.page.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getManuscript, getRppDoi } from '../../../../manuscripts';
+import { config } from '../../../../config';
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
Can this be restructured to return errors early so the happy path is the final return, rather than multi-level conditional nesting?